.gitignore

```diff
@@ -1,40 +1,44 @@
-/UnityProject/Assets/Photon\ Unity\ Networking/
-/UnityProject/Assets/PhotonChatApi
-/UnityProject/[Ll]ibrary/
-/UnityProject/[Tt]emp/
-/UnityProject/[Oo]bj/
-/UnityProject/[Bb]uild/
-/UnityProject/[Bb]uilds/
-/UnityProject/[Ll]ogs/
-/UnityProject/Assets/AssetStoreTools*
-/UnityProject/Assets/StreamingAssets/chatLogs*
-/UnityProject/Assets/InitTestScene*
-/UnityProject/Assets/MHLab*
-/UnityProject/PATCH*
-/UnityProject/.idea/
-/UnityProject/.vs/
-.vs
```

Comment:
isn't this the visual studio stuff?

Reply:
These were duplicates and should be covered by "IDE" section

Reply:
ok cool